both/methods.js

```diff
@@ -65,6 +65,11 @@ setSolveTime = function(resultId, solveIndex, solveTime) {
   RoundSorter.addRoundToSort(result.roundId);
 };
 
+function getTodayDateNoTime() {
+  var today = moment();
+  return new Date(Date.UTC(today.year(), today.month(), today.date()));
```

Comment:
Oh darn! Good thinking. You're right, we need to do this client side, because "today" may be different for the server and the client. I completely missed your point, sorry!

This isn't the biggest deal ever, as organizers are almost certainly going to change the date, but it would be nice to get this right. I'll make the change.